### PR TITLE
Adding Documentation on PypeIt conda-forge package

### PR DESCRIPTION
**Pull request by finagle29**
I'm not very experienced at making/editing .rst docs, so please point out any problems and places for improvement/going into more or less detail.

### CODE REVIEW

File: doc/dev/conda_forge.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+1. Fork `conda-forge/pypeit-feedstock <https://github.com/conda-forge/pypeit-feedstock>`_.
+2. Install required tools
+
+.. code-block bash::
+
+    conda install -c conda-forge conda-smithy
+
+3. Now manually update the version field in recipe/meta.yaml
+4. Update the source > sha256 hash in recipe/meta.yaml with the sha256 hash available on
```
Comment (profxj) [inline doc/dev/conda_forge.rst:16]: Not sure how `source > sha256` is intended to render.  Or if it all.
Please clarify

File: doc/dev/conda_forge.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+   `pypi <https://pypi.org/project/pypeit/#files>`_
+5. Reset the build > number to 0 in recipe/meta.yaml
```
Comment (profxj) [inline doc/dev/conda_forge.rst:18]: Same with `build > number`

**Review by profxj — APPROVED** [review]
Just a minor comment on some syntax.
It might be correct as is.  Not sure.

Am otherwise approving.

**Review by kbwestfall — APPROVED** [review]
Thanks @finagle29 !  I have the same comments as @profxj .  Please clarify and then feel free to merge.

File: doc/dev/conda_forge.rst (hunk 1 above)
Comment (finagle29) [reply doc/dev/conda_forge.rst]: I didn't intend `source > sha256` wasn't intended to render in any special way but I think `source.sha256` might be clearer as it is intended to represent

```yaml
source:
  sha256: hash_goes_here
```
in the `recipe/meta.yaml` file

File: doc/dev/conda_forge.rst (hunk 2 above)
Comment (finagle29) [reply doc/dev/conda_forge.rst]: same as above

File: doc/dev/conda_forge.rst (hunk 1 above)
Comment (profxj) [reply doc/dev/conda_forge.rst]: Got it!

In that case made do as you have done here but in .rst

You can use `::` to end a line and then indent the stuff below
to have it render sort of like what you have.  Albeit without color.

Like what we have in a doc like this:

https://github.com/pypeit/PypeIt/blob/master/doc/bias_dark.rst

Comment (finagle29) [reply doc/dev/conda_forge.rst]: okay, I tested building the docs locally and the code blocks showed up right

**Comment by profxj** [comment]
great!  feel free to merge when the Travis passes

**Comment by finagle29** [comment]
merged!

**Pull request merged**